aliSimple_test.go

```diff
@@ -34,8 +35,14 @@ func Test_AliClintReq(t *testing.T) {
 	str := cl0.reqMapToStr()
 	t.Log("map to str:" + str + "\n")
 	str = cl0.reqStrToSign(str, "GET")
+
+	// validate sign string from doc: https://help.aliyun.com/document_detail/29747.html#:~:text=%E9%82%A3%E4%B9%88-,stringtosign
+	if str != "GET&%2F&AccessKeyId%3Dtestid&Action%3DDescribeDomainRecords&DomainName%3Dexample.com&Format%3DXML&SignatureMethod%3DHMAC-SHA1&SignatureNonce%3Df59ed6a9-83fc-473b-9cc6-99c95df3856e&SignatureVersion%3D1.0&Timestamp%3D2016-03-24T16%253A41%253A54Z&Version%3D2015-01-09" {
```

Comment:
matching the documentation https://help.aliyun.com/document_detail/29747.html#:~:text=%E9%82%A3%E4%B9%88-,stringtosign